plugins/completion/cmp/cmp-helpers.nix

```diff
@@ -20,6 +20,8 @@ with lib; rec {
       maintainers = [maintainers.GaetanLepage];
     };
 
+  # WARNING: When adding a source, make sure to append the corresponding test in
+  # tests/test-sources/plugins/completion/cmp-all-sources.nix
   pluginAndSourceNames = {
```

Comment:
We could maybe put this in a separate file, and `import` it here & in the tests, so that we can automatically enable all sources